gcn/notices/icecube/LVK_nu_track_search.example.json

```
@@ -0,0 +1,47 @@
+{
+  "$schema": "https://gcn.nasa.gov/schema/gcn/notices/icecube/LVK_nu_track_search.schema.json",
+  "type": "IceCube LVK Alert Nu Track Search",
+  "ref_ID": "MS230427r",
+  "alert_type": "initial",
+  "record_number": 2,
```

```suggestion
  "reference": {"gcn.notices.LVK.alert": 4642 },
```
Apologies for confusion, we had discussion over it. These fields can create confusion for the data-base (in future) like alert_type of neutrino (parent instrument) or LVK (follow-up or counterpart). So it would be useful to use "reference" field existing in followUp core-schema that will direct to LVK notice. The alert_type and record_number is preserved for parent instrument like IceCube here.
We are working on actual reference name for LVK schema.  @dakota002

---

Thanks, will add a place holder for this, since I don't think it's there yet (correct?).  A question:  will this refence only be added to to the GCN-Kafka based notices for LVK or to the GCN classic as well.   Addtitionally, core/FollowUp.schema has this as "ref"

---

Ah, I see PR#90 now....can updated once merged